lib/kurtosis_service/kurtosis_service.go

```diff
@@ -21,35 +21,46 @@ const (
 	registrationRetryDurationSeconds = 60
 	regularOperationRetryDurationSeconds = 10
 
+	// Constants for making RPC calls to the Kurtosis API
 	kurtosisServiceStruct = "KurtosisService"
 	addServiceMethod = kurtosisServiceStruct + ".AddService"
 	removeServiceMethod = kurtosisServiceStruct + ".RemoveService"
 	registerTestExecutionMethod = kurtosisServiceStruct + ".RegisterTestExecution"
 )
 
-type KurtosisService struct {
+// Extracted as an interface for test mocking
```

Comment:
can we be explicit here in the documentation this interface/default struct is representing the Kurtosis API Container itself?